mlir/include/mlir/Dialect/Tensor/Transforms/Transforms.h

```diff
@@ -30,6 +30,24 @@ namespace tensor {
 FailureOr<TilingResult> replaceExtractSliceWithTiledProducer(
     OpBuilder &builder, tensor::ExtractSliceOp sliceOp, OpResult producerOp);
 
+/// Pattern to swap an `tensor.insert_slice` with its consumer when the
```

Comment:
Nit: Not a pattern (but I see I write "Pattern" above). Could you fix and make it a "Method"

Reply:
Doesn't look updated.